app/views/users/_user.html.erb

```diff
@@ -48,7 +49,8 @@
 
         <% end %>
       </div>
-
+      
+       <% if current_user == user %>
```

Comment:
This is nice! I didn't even think to hide other user's pending links. Great testing.

Reply:
thanks!

Reply:
You can use a policy here similarly to _photo and _comment, so we have all the authorization logic in one place.